CodeGeneration/Sources/SyntaxSupport/CommonNodes.swift

```diff
@@ -79,6 +79,27 @@ public let COMMON_NODES: [Node] = [
     ]
   ),
 
+  Node(
+    kind: .thrownType,
```

Comment:
The more consistent naming of this node would be `thrownTypeClause` in my opinion since it’s more than just the type (it also has parentheses).